src/Resource.php

```diff
@@ -54,8 +65,20 @@ public static function apiConnection($method, $url, $attributes) {
             'debug' => Config::getDebug()
         ];
         $res = $client->request($method, $url, $query);
+
+        if (env('MOXI_LOG_REQUESTS', false)) {
+            Log::channel('importJobs')->info(__METHOD__, ['headers' => $res->getHeaders()]);
+
+            $log = new MoxiImportLog();
```

Comment:
Shouldn't we also log the session key to be able to do some grouping? And maybe the type (listing/contact/etc) or endpoint